lightning/src/ln/channel.rs

```diff
@@ -1547,6 +1550,14 @@ pub(super) struct ChannelContext<SP: Deref> where SP::Target: SignerProvider {
 	/// If we can't release a [`ChannelMonitorUpdate`] until some external action completes, we
 	/// store it here and only release it to the `ChannelManager` once it asks for it.
 	blocked_monitor_updates: Vec<PendingChannelMonitorUpdate>,
+
+	/// Using this flag will prevent the funding transaction from being broadcasted 
+	/// and will allow the user to manually broadcast it.
+	///
+	/// The funding transaction can be accessed through the [`Event::FundingSigned`] event.
+	///
+	/// [`Event::FundingSigned`]: crate::events::Event::FundingSigned
+	manually_broadcast_outbound_channels: Option<()>,
```

Comment:
The `Option<()>` is mainly because other variables `is_batch_funding` and `local_initiated_shutdown` are defined with `Option` so I was trying to follow that.